container/build_tar.py

```diff
@@ -271,7 +271,21 @@ def add_deb(self, deb):
       while current:
         parts = current.filename.split(".")
         name = parts[0]
-        ext = '.'.join(parts[1:])
+        if parts[-1].lower() == 'xz':
+          try:
```

Comment:
any reason why you can't just reuse _xz_decompress here?

Reply:
No, that makes sense. Ill change it to support backports for python 2.7.